### ISSUE TITLE
Add a 'msg reply target' placeholder

### ISSUE THREAD

**Issue by Plum1111** [kangarko/ChatControl#741]
I think it would be a nice addition to add a placeholderapi compatible placeholder that shows who you currently will reply to via /reply. It could display 'none' if there is nobody who you will reply to and just the player name when you do have someone to reply to. A name that might work for it is `{chatcontrolred_reply_target}` (or `%chatcontrolred_reply_target%`) .
 
**Use cases** 

- It can be used from an admin perspective via DiscordSRV alerts. Currently if you want to log all privates messages to discord via alerts, it works very well with /msg and similar because both names are present so it looks like: `[Player1 -> Player2] hi` but when it is done via /reply it looks like `[Player2 reply] hey` and is hard to follow. The reply placeholder could be substituted in and they would be the same.

- It can also be used from a player perspective in a scoreboard down the side or in a custom command so people can remember who they are replying to easier. Some people need to deal with a lot of people and they might sometimes get confused as to who they are replying to.

**What would happen if we won't implement this feature? How's your server being affected?**

It's mostly a convenience thing for discord alerts in my case. It gets increasingly difficult to follow them as more people message at once. I understand if you don't add it but there would be a lot less backtracking.

**Comment by kangarko** [comment]
Thanks will be added! Great suggestion btw <3
